eidpauth/src/test/java/uk/ac/ucl/eidp/auth/jaas/SingleFactorLoginModuleNgTest.java

```diff
@@ -0,0 +1,109 @@
+package uk.ac.ucl.eidp.auth.jaas;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.spy;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Map;
+import javax.security.auth.Subject;
+import javax.security.auth.callback.Callback;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.callback.UnsupportedCallbackException;
+
+
+
+
+/**
+ * Unit tests for SingleFactorLoginModule.
+ * @author rebmdgu
+ */
+public class SingleFactorLoginModuleNgTest {
+  
+  public SingleFactorLoginModuleNgTest() {
```

Comment:
\[PMD\] INFO: Avoid unnecessary constructors \- the compiler will generate these for you